packages/flutter/lib/src/material/drawer.dart

```diff
@@ -118,14 +133,20 @@ class DrawerController extends StatefulWidget {
   const DrawerController({
     GlobalKey key,
     @required this.child,
-  }) : assert(child != null),
+    @required this.alignment,
+  }) : assert(child != null || alignment != null),
```

Comment:
Do you really mean `||` here?  I would have expected

```dart
assert(child != null),
assert(alignment != null),
```

Which is the "and" rather than "or" of these conditions.

Reply:
Fixed